### PR TITLE
Use https for cdn to take advantage of http/2.

### PR DESCRIPTION
**Pull request by ahmedre**

### CODE REVIEW

**Comment by mmahalwy** [comment]
hey, our functional tests are working. woohoo!

**Comment by mmahalwy** [comment]
lgtm

**Comment by mmahalwy** [comment]
ill fix tests later

File: server/routes.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function(server) {
   });
 
   server.get(/^\/(images|fonts)\/.*/, function(req, res) {
```
Comment (azizur) [inline server/routes.js]: For performance reasons the `301` redirect should be done in web server level (ie: `apache`/`nginx`). Where these required can be cached for ling time will lead to better performance on the application layer.

File: src/scripts/components/Loader.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Loader extends React.Component {
       <div className="loading" ng-hide="currentSurah.ayahs">
         <div className="row">
           <div className="col-md-6 col-md-offset-3 text-center">
-            <img src="//quran-1f14.kxcdn.com/images/loading.gif" />
+            <img src="https://quran-1f14.kxcdn.com/images/loading.gif" />
```
Comment (azizur) [inline src/scripts/components/Loader.js]: With-in the application we should not really care about the protocols. This should be left to web servers to deal with.

File: server/routes.js (hunk 1 above)
Comment (ahmedre) [inline server/routes.js]: excellent point, agreed.

File: src/scripts/components/Loader.js (hunk 2 above)
Comment (ahmedre) [inline src/scripts/components/Loader.js]: makes sense

**Comment by ahmedre** [comment]
abandoning this for now, will do this on a clean slate taking into account suggestions by @azizur.

File: server/routes.js (hunk 1 above)
Comment (mmahalwy) [inline server/routes.js]: That'd be an overkill. This is a backup incase a request for an image is sent directly to our servers. We should never have that.

File: src/scripts/components/Loader.js (hunk 2 above)
Comment (mmahalwy) [inline src/scripts/components/Loader.js]: It'll change when I add `url-loader`

**Pull request closed without merging**